### PR TITLE
Document how to run the ConsoleLauncher with classpath wildcards

### PR DESCRIPTION
**Pull request by mbucc**
## Overview

This enables you to use the compact "jar star" pattern in your classpath.
Partially addresses https://github.com/junit-team/junit5/issues/2788

---

I hereby agree to the terms of the [JUnit Contributor License Agreement](https://github.com/junit-team/junit5/blob/002a0052926ddee57cf90580fa49bc37e5a72427/CONTRIBUTING.md#junit-contributor-license-agreement).

---

### Definition of Done

- [x] There are no TODOs left in the code
- [x] Method [preconditions](https://junit.org/junit5/docs/snapshot/api/org.junit.platform.commons/org/junit/platform/commons/util/Preconditions.html) are checked and documented in the method's Javadoc
- [x] [Coding conventions](https://github.com/junit-team/junit5/blob/HEAD/CONTRIBUTING.md#coding-conventions) (e.g. for logging) have been followed
- [x] Change is covered by [automated tests](https://github.com/junit-team/junit5/blob/HEAD/CONTRIBUTING.md#tests) including corner cases, errors, and exception handling
- [x] Public API has [Javadoc](https://github.com/junit-team/junit5/blob/HEAD/CONTRIBUTING.md#javadoc) and [`@API` annotations](https://apiguardian-team.github.io/apiguardian/docs/current/api/org/apiguardian/api/API.html)
- [x] Change is documented in the [User Guide](https://junit.org/junit5/docs/snapshot/user-guide/) and [Release Notes](https://junit.org/junit5/docs/snapshot/user-guide/#release-notes)
- [x] All [continuous integration builds](https://github.com/junit-team/junit5#continuous-integration-builds) pass

### CODE REVIEW

**Review by sormuras — CHANGES_REQUESTED** [review]
image

I guess, your changes will end up in two lines reading "Exit code". Did you generate the User Guide HTML file locally (via: `gradlew asciidoctor`) and viewed it (`documentation/build/docs/asciidoc/user-guide/index.html`) with a browser?

**Comment by mbucc** [comment]
> Did you generate the User Guide HTML file locally (via: `gradlew asciidoctor`) and viewed it (`documentation/build/docs/asciidoc/user-guide/index.html`) with a browser?

No.  I copied the asciidoc from the prior run statement and viewed the result in browser.  I noticed the Exit Code in the output, but saw the same for the previous run command and so thought it was correct.

I'm generating a local User Guide HTML now, and will update the patch.

**Review by sormuras — APPROVED** [review]
LGTM

File: documentation/src/docs/asciidoc/user-guide/running-tests.adoc [hunk 1]
```diff
@@ -563,7 +563,18 @@ Test run finished after 64 ms
 [         0 tests failed          ]
 ----
 
+You can also https://docs.oracle.com/javase/tutorial/deployment/jar/run.html[run] the
+standalone `ConsoleLauncher` as shown below
+(for example, to include all jars in directory):
```
Comment (marcphilipp) [inline documentation/src/docs/asciidoc/user-guide/running-tests.adoc:568]:
```suggestion
(for example, to include all jars in a directory):
```

File: documentation/src/docs/asciidoc/user-guide/running-tests.adoc [hunk 2]
```diff
@@ -563,7 +563,18 @@ Test run finished after 64 ms
 [         0 tests failed          ]
 ----
 
+You can also https://docs.oracle.com/javase/tutorial/deployment/jar/run.html[run] the
```
Comment (marcphilipp) [inline documentation/src/docs/asciidoc/user-guide/running-tests.adoc:566]: Why does this link to the docs for using `-jar`?

Comment (mbucc) [reply documentation/src/docs/asciidoc/user-guide/running-tests.adoc:566]: It should not.  I mistakenly copied it from the existing "you can run" text above (line 532).

**Review by marcphilipp — APPROVED** [review]

**Pull request merged**